doc/CONTRIBUTING.md

```diff
@@ -102,6 +102,20 @@ Good commit message subject :
 Document how to install the project
 ```
 
+Also, when you specify what you did avoid commit message subjects with "Fix bug
+in such and such feature". Saying you are fixing something implies the previous
+implementation was wrong and yours is right, which might not be even true.
+Instead, humbly say what you did technically and **then** explain how this is
```

Comment:
> or you think it was

This is key. Sometimes you think it was, but the commit message for the commit that introduced what you are fixing is so imprecise that you don't see how the piece of code actually is useful, until you hit a bug.

Plus, the shortlog is what your doing, and thus, stating unquestionable facts (unlike I'm probalby fixing a bug). Fixing a bug is *why* you are doing it, hence it should be in the § below. Plus, while it is strongly recommended to refer to an existing issue, you should always at least sum it up (still not relying on a web application to explain what you do). `Fixes #666` is a plus. It is a big one, but it should be considered just a plus.